src/buildMatchSnapshot.js

```diff
@@ -73,10 +77,10 @@ const buildMatchSnapshot = (utils, parseArgs) => {
     //
     // Note: we only test the top-level data for cusotm serializers
     // (jest might look for them recursively).
-    const logCustomSerializerWarning = obj !== toMatch // we cloned
-      && getSerializers().some(it => it.test(obj)); // has custom serializers
+    const logCustomSerializerWarning =
+      obj !== toMatch && getSerializers().some(it => it.test(obj)); // we cloned // has custom serializers
```

Comment:
Can you fix up the comment here (rewording as necessary)? It got a little weird with prettier

Reply:
Missed that! Not totally sure about the fix:
`// was  cloned & has custom serializers`
?

Reply:
Yeah, that works. Exact wording doesn't matter too much; just the `// ... // ....` was weird.